analyzers/rspec/cs/S2222_c#.json

```diff
@@ -0,0 +1,23 @@
+{
+  "title": "Locks should be released",
+  "type": "BUG",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "20min"
+  },
+  "tags": [
+    "cwe",
+    "multi-threading"
+  ],
+  "defaultSeverity": "Critical",
+  "ruleSpecification": "RSPEC-2222",
+  "sqKey": "S2222",
+  "scope": "All",
```

Comment:
No other SE rule runs on Test projects. Let's change this one as well (it actually breaks `SymbolicExecutionRunner` assumptions)

Reply:
Why wouldn't we want to run on Test projects in the future?

Reply:
This particular rule doesn't make sense for test projects.
We can reconsider other rules in the future.